app/views/jobs/index.html.erb

```diff
@@ -8,7 +8,7 @@ This page shows you the status of current and recent autograding jobs.
 
 <h2>Currently Running Jobs</h2>
 
-<% if @running_jobs == [] then %>
+<% if !@running_jobs or @running_jobs == [] then %>
```

Comment:
I think @running_jobs.blank? would be better:
http://stackoverflow.com/questions/885414/a-concise-explanation-of-nil-v-empty-v-blank-in-ruby-on-rails